pyramid_oereb/lib/records/geometry.py

```diff
@@ -132,45 +132,44 @@ def calculate(self, real_estate, min_length, min_area, length_unit, area_unit):
         point_types = geometry_types.get('point').get('types')
         if self.published:
             result = self._extract_collection(self.geom.intersection(real_estate.limit))
-            if not result.is_empty:
-                if self.geom.type not in point_types + line_types + polygon_types:
-                    supported_types = ', '.join(point_types + line_types + polygon_types)
-                    raise AttributeError(
-                        u'The passed geometry is not supported: {type}. It should be one of: {types}'.format(
-                            type=self.geom.type, types=supported_types
-                        )
+            # differentiate between Points and MultiPoint
```

Comment:
Maybe you could add an additional comment with a hint for the needed `if not result.is_empty` on updating Shapely again.